doc/conf.py

```diff
@@ -30,7 +30,7 @@
 
 if "conda" in sys.executable:
     print("conda environment:")
-    subprocess.run(["conda", "list"])
+    subprocess.run([os.environ.get("CONDA_EXE", "conda"), "list"])
```

Comment:
would it be better to call it the same way as `pip`?
```suggestion
    subprocess.run([sys.executable, "-m", "conda", "list"])
```

Reply:
As far as I know, executing *conda* as a module will not work generally. My understanding is that typically only the base environment contains the *conda* package and may import it, while other environments will find no module named "conda".

Reply:
That's strange, I was thinking that `conda` would work the same on all platforms, and I can import `conda` just fine in all my environments. I can't verify if that's also true for windows, though. What version of `conda` are you using?

Reply:
`pip` and `conda` are fundamentally different here. `pip` only works with the Python for which it is installed. This is why one uses `python -m pip` to ensure that you get a `pip` that matches the currently active `python` executable.

`conda` manages multiple environments, so it's useful (and really, normal) that the `conda` executable isn't coming from the same environment as `python`.  The only concern here is either making sure `python` points to what's desired or that `conda` is working on the right environment--or explicitly passing the environment name to conda commands.

Reply:
Thanks for describing what motivates listing the *conda* environment and for noting the high-level differences between the *pip* and *conda* commands. My reading of our comments is that the commit does as desired, in that "conf.py" calls the *conda* command available in the shell (looked up through `CONDA_EXE` with fall-back to "conda"). I'll resolve the conversation but welcome other comments.